app/components/keyword/keywordDirective.js

```diff
@@ -0,0 +1,19 @@
+(function() {
+    angular
+    .module('search_keyword_component', [])
+    .directive('keyword', function keyword() {
+        return {
+            scope: {
+                tag: '@',
+                remove: '=',
+                removefn: '='
```

Comment:
please use `&` when you want to call a function. 
do not pass the function in the scope.